Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN apt-get install -y -q netcat
 RUN mkdir -p $GOPATH/src/github.com/oysterprotocol/brokernode
 WORKDIR $GOPATH/src/github.com/oysterprotocol/brokernode
 
-RUN go get -u github.com/gobuffalo/buffalo/buffalo
+RUN go get -u -v github.com/gobuffalo/buffalo/buffalo
```

Comment:
Was this intentional?  I'm not disagreeing, just checking.

Reply:
Yeah, it just prints more stuff so we can see what's going on.